tests/CMakeLists.txt

```diff
@@ -5,6 +5,7 @@ if(${run_unittests})
     add_subdirectory(agenttime_ut)
     add_subdirectory(azure_base64_ut)
     add_subdirectory(buffer_ut)
+    add_subdirectory(constbuffer_ut)
```

Comment:
>constbuffer_ut [](start = 21, length = 14)

interesting, wasn't even building

Reply:
Yep, missed it when moving things the first time.